pco_cli/src/input/audio.rs

```diff
@@ -11,12 +11,26 @@ pub fn get_wav_field(path: &Path) -> Result<Option<Field>> {
   // this is excessively slow, but easy for now
   let mut file = File::open(path)?;
   let (header, _) = wav::read(&mut file)?;
-  let dtype = match header.bytes_per_sample {
-    1 | 2 => Ok(DataType::Int16),
-    3 => Ok(DataType::Int32),
-    4 => Ok(DataType::Float32),
+  let dtype = match header.audio_format {
+    wav::WAV_FORMAT_PCM => match header.bits_per_sample {
+      8 | 16 => Ok(DataType::Int16),
+      24 | 32 => Ok(DataType::Int32),
+      _ => Err(anyhow!(
+        "unsupported bit depth for WAV_FORMAT_PCM: {}",
+        header.bits_per_sample
+      )),
+    },
+    wav::WAV_FORMAT_IEEE_FLOAT => match header.bits_per_sample {
+      16 => Ok(DataType::Float16),
```

Comment:
Is this a real possibility? The `wav` crate doesn't allow decoding 16 bit floats (see `get_array` below), so I think we should only handle the int16, int32, and float32 cases.

Reply:
Ok, I'll remove `16 => Ok(DataType::Float16),`.